### PR TITLE
Add/feedback page

### PR DESCRIPTION
**Pull request by MistahSanta**
Add basic feedback page that send the form data to supabase database and added  a popup after form is submitted.

### CODE REVIEW

**Comment by MistahSanta** [comment]
issue with merging 😭

**Comment by nl32** [comment]
did you rebase? commit history seems a bit off

File: src/app/feedback/page.tsx [hunk 1]
```diff
@@ -1,6 +1,10 @@
+import React from 'react';
 import Header from '@src/components/BaseHeader';
 import { type Metadata } from 'next';
-import Image from 'next/image';
+import  Form  from '@src/app/feedback/Form';
+
+//import { useState } from 'react';
```
Comment (RubenOlano) [inline src/app/feedback/page.tsx:6]: Delete this commented line

File: src/server/db/schema/forms.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import {  sql } from "drizzle-orm";
+import { pgTable, text, timestamp, integer } from "drizzle-orm/pg-core"
+
+export const forms = pgTable("formSubmission", {
```
Comment (nl32) [inline src/server/db/schema/forms.ts:4]: formSubmission might be a bit too vague should probably say it's for the feedback form, also make sure to generate a db migration with `drizzle:generate`.

**Review by RubenOlano — CHANGES_REQUESTED** [review]
Looks pretty good, just one line to delete. I would also add a disabled state on the form whenever a user has already submitted the form. Just so that you can't accidentally spam the form

**Review by nl32 — CHANGES_REQUESTED** [review]
Feedback form looks clean, though the nebula logo might be a bit redundant, at least not on mobile. no merge issues 😅!

**Review by RubenOlano — APPROVED** [review]
LGTM

**Review by nl32 — APPROVED** [review]
LGTM

**Pull request merged**